### DISCUSSION TITLE
Does cloudbet API support Python programming

### DISCUSSION THREAD

**Discussion by donnolic** [cloudbet/docs#93] [General]
I have done some research and read many documents on cloudbet API but can't find anyone written in Python. Please can anyone help me on this. Any API document in python. Thanks

**Comment by kgravenreuth** [comment]
Hi @donnolic 
While we don't have a Python library or SDK for the Cloudbet API yet, we would encourage you to look at existing documentation and tutorials to get an idea of the responses that you should be expecting here. Furthermore, please note that the response is JSON encoded, which might need some handling on your part.

1. [How to build a betting bot with the Cloudbet API](https://web.archive.org/web/20230926154511/https://bitedge.com/blog/how-to-build-a-betting-bot-with-the-cloudbet-api/) -> This is based on Python. Note that the [original blog post](https://bitedge.com/blog/how-to-build-a-betting-bot-with-the-cloudbet-api/) was removed by this third party site, so this is a Web Archive link.
2. [Sports Betting API Tutorial With Golang](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang?utm_source=github.com&utm_medium=community&utm_campaign=axis)


Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
